api/controllers/MerchStoreController.ts

```diff
@@ -197,18 +204,57 @@ export class MerchStoreController {
     return originalOrder;
   }
 
-  @Patch('/order')
-  async editMerchOrder(@Body() fulfillOrderRequest: FulfillMerchOrderRequest,
+  @Patch('/order/:uuid/pickup')
+  async editMerchOrderPickup(@Params() params: UuidParam,
+    @Body() editOrderRequest: EditMerchOrderPickupRequest,
     @AuthenticatedUser() user: UserModel): Promise<EditMerchOrderResponse> {
-    if (!PermissionsService.canFulfillMerchOrders(user)) throw new ForbiddenError();
+    if (!PermissionsService.canAccessMerchStore(user)) throw new ForbiddenError();
+    await this.merchStoreService.editMerchOrderPickup(params.uuid, editOrderRequest.pickupEvent, user);
+    return { error: null };
+  }
+
+  @Post('/order/:uuid/cancel')
+  async cancelMerchOrder(@Params() params: UuidParam, @AuthenticatedUser() user: UserModel) {
+    if (!PermissionsService.canAccessMerchStore(user)) throw new ForbiddenError();
```

Comment:
```suggestion
    if (!PermissionsService.canManageMerchOrders(user)) throw new ForbiddenError();
```

Reply:
Members can cancel their own orders, so they don't need 'manage merch order' permissions since that's something a store manager / store distributor would need